content/en/docs/concepts/security/service-accounts.md

```diff
@@ -0,0 +1,241 @@
+scenario where you have a service account and Pod in the `dev` namespace and
+you want your Pod to see Jobs running in the `maintenance` namespace. You could
+create a Role object that grants permissions to list Job objects. Then,
+you'd create a RoleBinding object in the `maintenance` namespace to bind the
+Role to the ServiceAccount object. Now, Pods in the `dev` namespace can list
+Job objects in the `maintenance` namespace using that service account.
+
+### Assign a ServiceAccount to a Pod {#assign-to-pod}
+
+To assign a ServiceAccount to a Pod, you set the `spec.serviceAccountName`
+field in the Pod specification. Kubernetes then automatically provides the
+credentials for that ServiceAccount to the Pod. In v1.22 and later, Kubernetes
+gets a short-lived, automatically rotating token using the `TokenRequest` API
+and mounts the token as a
+[projected volume](/docs/concepts/storage/projected-volumes/#serviceaccounttoken).
+If the Pod or the ServiceAccount is deleted, the token is no longer valid in the
+Kubernetes API.
+
+<!-- OK to remove this historical detail after Kubernetes 1.31 is released -->
+
+In versions earlier than 1.22, Kubernetes provides a long-lived, static token
+to the Pod as a Secret.
+
+#### Manually retrieve ServiceAccount credentials {#get-a-token}
+
+If you need the credentials for a ServiceAccount to mount in a non-standard
+location, or for an audience that isn't the API server, use one of the
+following methods:
+
+* [TokenRequest API](/docs/reference/kubernetes-api/authentication-resources/token-request-v1/)
+  (recommended): Request a short-lived service account token using your
+  *application code*. The token expires automatically and can rotate upon
+  expiration.
+* [Token Volume Projection](/docs/tasks/configure-pod-container/configure-service-account/#service-account-token-volume-projection)
+  (recommended): In Kubernetes v1.20 and later, use the Pod specification to
+  tell the kubelet to add the service account token to the Pod as a
+  *projected volume*. Projected tokens expire automatically, and the kubelet
+  rotates the token before it expires. The ServiceAccount token is also
+  invalidated in the Kubernetes API when the Pod or the ServiceAccount is
+  deleted.
+* [Service Account Token Secrets](/docs/tasks/configure-pod-container/configure-service-account/#manually-create-a-service-account-api-token)
+  (not recommended): You can mount service account tokens as Kubernetes
+  Secrets in Pods. These tokens don't expire and don't rotate. This method
+  is not recommended, especially at scale, because of the risks associated
+  with static, long-lived credentials. In Kubernetes v1.24 and later, the
+  [LegacyServiceAccountTokenNoAutoGeneration feature gate](/docs/reference/command-line-tools-reference/feature-gates/#feature-gates-for-graduated-or-deprecated-features)
+  prevents Kubernetes from automatically creating these tokens for
+  ServiceAccounts.
```

Comment:
should we mention `kubectl create token` here, or update any of the linked docs?

Reply:
IMO: that can happen in a follow up PR.

Reply:
That's fair.

I should just add that if we wanted to keep it in this doc, the right place would be around the TokenRequest API as that's used as the backend for the command.

Reply:
Ack, thank you